src/middlewares/validation/validation.js

```diff
@@ -89,3 +91,35 @@ export const ValidateEmptySignUpBodyProperty = async (req, res, next) => {
   }
   return next();
 };
+
+export const validationForSignIn = (req, res, next) => {
+  let {
+    email, password
+  } = req.body;
+
+  if (req.body.email) {
+    email = email.trim();
+  }
+  if (req.body.password) {
+    password = password.trim();
+  }
+
+  const signInBody = {
```

Comment:
Please implement the following feedback:
`1.`  I noticed you're `trimming` the `email` for validation but you're still passing the `untrimmed` email to the controller. Please replace the `untrimmed email` with the `trimmed email` on the `req.body`.

Reply:
Alright

Reply:
you've not implemented the feedback. Please read the comment carefully.
What you did was `trim` the `email` in the validation. Please also trim it in the controller before running `findUserInUsersDb(email)`;

Reply:
Alright, Fixed